src/main/resources/swagger.api/schemas/dcbPickup.yaml

```diff
@@ -0,0 +1,16 @@
+DcbPickup:
+  description: Pickup Location metadata required for the pickup service point
+  type: object
+  properties:
+    libraryName:
+      description: The name of the pickup library
+      type: string
+    libraryCode:
+      description: The code which identifies the pickup library
+      type: string
+    servicePointId:
+      description: The code which identifies the pickup service point
```

Comment:
I think it is better to be more specific here. Please mention something like UUID of the service point.